Extending/Section-Trees/tree-actions.md

```diff
@@ -82,5 +82,11 @@ If you have created a package using a custom tree action, please consider provid
 |Umbraco.Deploy.UI.Actions.ActionDeployTreeRestore|deployTreeRestore|Ψ|True|
 |Umbraco.Deploy.UI.Actions.ActionPartialRestore|deployPartialRestore|Ø|True|
 |Umbraco.Deploy.UI.Actions.ActionQueueForTransfer|deployQueueForTransfer|TBC|True|
+|Jumoo.TranslationManager.Core.Actions.ActionTranslate|translate|5|True|
+|Jumoo.TranslationManager.Core.Actions.ActionManageTranslation|manageTranslations|Ť|True|
+|uSync.Publisher.Actions.PushToServer|pushContent|>|True|
+|uSync.Publisher.Actions.PullFromServer|oullContent|<|True|
```

Comment:
@KevinJump Just to be sure: `oullContent` isn't a typo in this PR, but actually used in your package?

Reply:
yeah sorry, thats a typo with me, copying it wrong :)

Reply:
the alias is "pullContent"

Reply:
Have updated - thanks.